src/reference/asciidoc/jdbc.adoc

```diff
@@ -480,6 +480,9 @@ Instead, if possible, consider using either JMS- or AMQP-backed channels instead
 For further reference, see the following resource:
 
 * https://mikehadlow.blogspot.com/2012/04/database-as-queue-anti-pattern.html[The Database As Queue Anti-Pattern].
+
+If you are still planning to use your database as a queue, consider using PostgreSQL and its notification
+mechanism which is described in a subsequent section.
```

Comment:
Great! Can you, please, follow the rule of `One sentence per line`: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line ?